guide/dep-manip.md

```diff
@@ -91,36 +82,6 @@ The lookup REST endpoint should follow:
 </tr>
```

Comment:
I Think we might be missing a `</td>` tag here.

Reply:
I count 2 open td and 2 close? Where please?

Reply:
Nevermind, it seems that 2 lines were folded on the left side 89--90 and it was there.